### PR TITLE
Fix cauldron app name for releasing to codepush

### LINKED ISSUE #1843: ern code-push release doesn't recognize custom application name in cauldron
In cauldron.json file, I specified appName like below. But ern doesn't recognize this and takes as MyAppAndroid. And this leads to Codepush 404 error. 

Should resolve like My-Team/MyAppAndroid

```
{
  "name": "Cauldron",
  "nativeApps": [
    {
      "name": "MyApp",
      "platforms": [
        {
          "name": "android",
          "config": {
            "codePush": {
              "appName": "My-Team/MyAppAndroid"
            }
          }
        }
      ]
    }
  ],
  "schemaVersion": "3.0.0"
}


```

### PR DESCRIPTION
**Pull request by ozcanzaferayan**
Uses cauldron CodePush config app name when releasing to CodePush.

Fixes #1843

### CODE REVIEW

**Comment by friederbluemle** [comment]
Thanks for the PR @ozcanzaferayan :+1:

1. Can you please sign the CLA?
2. Any chance of adding a test case for the modified lines? Have a look around line 843 in `codepush-test.ts`: https://github.com/electrode-io/electrode-native/blob/master/ern-orchestrator/test/codepush-test.ts#L843

**Comment by ozcanzaferayan** [comment]
Thanks I signed CLA and added test for changes.

File: ern-orchestrator/test/codepush-test.ts [hunk 1]
```diff
@@ -850,6 +850,16 @@ describe('codepush', () => {
...
       expect(result).equal('app-android');
     });
+
+    it('should use the custom app name from cauldron if any', async () => {
+      sandbox.restore();
+      cauldronDoc = cloneFixture(fixtures.defaultCauldronCustomAppName);
+      const cauldronHelper = createCauldronHelper(cauldronDoc);
+      sandbox.stub(cauldronApi, 'getActiveCauldron').resolves(cauldronHelper);
+      const result = await sut.getCodePushAppName(testAndroid1780Descriptor);
+      console.log(result);
```
Comment (friederbluemle) [inline ern-orchestrator/test/codepush-test.ts:860]: Remove console.log please

**Review by friederbluemle — COMMENTED** [review]
Looks great! One minor comment, and this should be good to merge!
Can you also squash your commits please?

**Review by friederbluemle — APPROVED** [review]
I've squashed the two commits, and removed the console.log statement.

Thanks for your contribution @ozcanzaferayan :+1:

**Comment by ozcanzaferayan** [comment]
Thanks so much. In past weeks we are worked on Hackathon project and not have time for this PR. In future i will pay attention this consoleLog and squash issues. Thanks again.

**Pull request merged**